version/version.go

```diff
@@ -25,7 +25,9 @@ import (
 )
 
 var (
-	Version = "2.1.0-alpha.0+git"
+	// MinVersion of etcd this version can work with
+	MinVersion = "2.0.0"
```

Comment:
yea... i force patch to be zero in the code and here.
https://github.com/coreos/etcd/pull/2718/files#diff-971f6af906b31cc86efd71aff56aa96bR898

I probably should just leave it to be major.minor here.